tests/e2e-tests/specs/backend/all-products.test.js

```diff
@@ -15,6 +15,13 @@ const block = {
 	class: '.wc-block-all-products',
 };
 
+/**
+ * @todo: write helpers to simplify version and feature gating tests.
+ */
+if ( process.env.WP_VERSION < 5.3 )
+	// eslint-disable-next-line jest/no-focused-tests
+	test.only( 'skipping all other things', () => {} );
```

Comment:
This skipping message should probably include a reason so it's obvious when viewing output.

Reply:
I plan to abstract those later in a follow up PR with better messages.